server/index.js

```diff
@@ -108,6 +109,7 @@ const nomisPushService = createNomisPushService(nomisClientBuilder, signInServic
 const notificationJobs = createNotificationJobs(reminderService, signInService)
 const jobSchedulerService = createJobSchedulerService(dbLockingClient, configClient, notificationJobs)
 const lduService = createLduService(deliusClient, activeLduClient)
+const functionalMailboxService = new FunctionalMailboxService(deliusClient, probationTeamsClient)
 const licenceSearchService = createLicenceSearchService(licenceClient, signInService, nomisClientBuilder)
```

Comment:
Um.  Perhaps creating a class was a bad idea - because it is inconsistent, and not really needed. I'm prepared to change this if the reviewer agrees...

Reply:
Using classes means that you don't need to define separate interfaces -  you can import the class as a type:

```ts
import  FunctionalMailboxService from '../../..'

export default function someThing(functionalMailboxService: FunctionalMailboxService)
```

Reply:
OK